src/utils/misc.ts

```diff
@@ -49,3 +49,29 @@ export function* splitIntoBatches<T>(list: T[], maxBatchSize: number): Generator
     yield list.slice(offset);
   }
 }
+
+export function unwrapData(data: { __kind: string; value?: string }) {
+  switch (data.__kind) {
+    case 'None':
+      return null;
+    case 'BlakeTwo256':
+    case 'Sha256':
+    case 'Keccak256':
+    case 'ShaThree256':
+      return Buffer.from(data.value!).toString('hex');
+    default: {
+      let unwrapped = '';
+      if (isHex(data.value)) {
+        unwrapped = decodeHex(data.value).toString('utf-8');
+      } else {
+        unwrapped = Buffer.from(data.value!).toString('utf-8');
+      }
+
+      return unwrapped.replace(/\u0000/g, '');
```

Comment:
Can you add a comment explaining what's being replaced?

Reply:
It removes the `null` character from the strings that comes from the blockchain. I'll add a comment